packages/expo-keep-awake/src/index.ts

```diff
@@ -28,10 +29,10 @@ export default class KeepAwake extends React.PureComponent {
   }
 }
 
-export function activate() {
-  ExpoKeepAwake.activate();
+export function activate(tag?: string) {
+  ExpoKeepAwake.activate(tag ? tag : ExpoKeepAwakeTag);
```

Comment:
Wouldn't this work too?

```suggestion
  ExpoKeepAwake.activate(tag || ExpoKeepAwakeTag);
```